libs/agora/genes/src/lib/components/gene-hero/gene-hero.component.html

```diff
@@ -29,22 +29,29 @@ <h4 class="gene-hero-biodomains-heading">Biological Domains</h4>
               </p>
             </div>
           }
+          @if (gene.druggability && gene.druggability.pharos_class) {
+            <div class="gene-hero-pharos">
+              <h4 class="gene-hero-pharos-heading">Pharos Class</h4>
+              <p>{{ gene.druggability.pharos_class.join(', ') }}</p>
+            </div>
+          }
           <div class="gene-hero-aliases">
             <h4 class="gene-hero-aliases-heading">Also known as</h4>
-            @if (getEnsemblUrl() !== '') {
-              <p>
+            @if (getEnsemblUrl()) {
```

Comment:
```suggestion
            @if (getEnsemblUrl(); as ensembleUrl) {
```

Reply:
Reference the result of the function and use it in the if block, instead of calling the getter twice.

Reply:
nice catch!